src/mesh/voxel_cloud.rs

```diff
@@ -14,6 +14,27 @@ use crate::plane::Plane;
 
 use super::{primitive, tools, Face, Mesh, NormalStrategy};
 
+/// Describes how should be empty (None) voxel be treated. Either as None or as
+/// a constant value. This is used for arithmetic functions, such as addition.
+#[allow(dead_code)]
+#[derive(Debug)]
+pub enum TreatEmpty {
+    AsNone,
+    AsValue(f32),
+}
+
+/// Selects falloff function for the falloff field computation. The parameter
```

Comment:
"for the distance field computation"? I searched, but I see nothing named falloff field.